.github/CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# All markdown files
+*.md @xmarcalx @kalyazin @pb8o @sudanl0
+
+# But not the ones in docs/
+docs/*.md
+
+# Except these specific ones
+docs/RELEASE_POLICY.md @xmarcalx @kalyazin @pb8o @sudanl0
```

Comment:
I do not think we need to specify just this policy file but we should include any policy file in this rule
I am not sure if the rule is case sensitive but if it is case sensitive i will simply define a rule
*policy*.md
*POLICY*.md

or in case it is case insensitive just one of the two. 

At the moment you have a bug that a file like kernel-policy.md would be ignored.

Reply:
Ahh, I thought we only had one policy file. Yes I think a wildcard rule like that will work (I don't think its case insensitive)

Reply:
NP, even if we had only one policy document at the moment, it would not have been very future proofing having such rule so we should think in case to something else.